### PR TITLE
Added ability for autodoc to check radiation level and administer rad-away if needed

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
SUMMARY: Features "Added ability for autodoc to check radiation level and administer rad-away if needed."

#### Purpose of change
Increase functionality of autodoc.

#### Describe the solution
Mostly copy-paste of `geiger` iuse function with some minor tweaks. Also added an ability to administer Prussian blue in case of radiation poisoning.

#### Describe alternatives you've considered
None.

#### Testing
Got radiated, went to autodoc, asked if it could check my rad level.

#### Additional context
None.

### CODE REVIEW

File: src/iexamine.cpp [hunk 1]
```diff
@@ -4904,6 +4907,27 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             break;
         }
 
+        case RAD_AWAY: {
+            patient.moves -= 500;
+            patient.add_msg_player_or_npc( m_info,
+                                           _( "The autodoc scanned you and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           _( "The autodoc scanned <npcname> and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           patient.get_rad(), patient.leak_level( "RADIOACTIVE" ) );
```
Comment (nexusmrsep) [inline src/iexamine.cpp:4915]: Perhaps (for immersion reasons) autodoc could suggest undergoing decontamination process `if ( patient.leak_level( "RADIOACTIVE" ) > 0 )` aka getting rid of radioactive item.

Comment (Night-Pryanik) [reply src/iexamine.cpp:4915]: Good idea, thanks.

File: src/iexamine.cpp [hunk 2]
```diff
@@ -4904,6 +4907,27 @@ void iexamine::autodoc( player &p, const tripoint &examp )
...
+            if( patient.get_rad() ) {
+                if( patient.has_effect( effect_pblue ) ) {
+                    patient.add_msg_player_or_npc( m_info,
+                                                   _( "The autodoc detected an anti-radiation drug in your bloodstream, so it decided not to administer another dose right now." ),
+                                                   _( "The autodoc detected an anti-radiation drug in <npcname>'s bloodstream, so it decided not to administer another dose right now." ) );
+                } else {
+                    add_msg( m_good,
```
Comment (BevapDin) [inline src/iexamine.cpp:4922]: Shouldn't this be `add_msg_player_or_npc` as well?

Comment (Night-Pryanik) [reply src/iexamine.cpp:4922]: The text is identical for both player and NPC, so I thought generic `add_msg` will suffice.

**Pull request merged**